clients/drcachesim/tools/invariant_checker.cpp

```diff
@@ -440,7 +458,8 @@ invariant_checker_t::parallel_shard_memref(void *shard_data, const memref_t &mem
                           TRACE_MARKER_TYPE_KERNEL_XFER)) ||
                     // We expect a gap on a window transition.
                     shard->window_transition_ ||
-                    shard->prev_instr_.instr.type == TRACE_TYPE_INSTR_SYSENTER,
+                    shard->prev_instr_.instr.type == TRACE_TYPE_INSTR_SYSENTER ||
```

Comment:
Why do we need this additional condition here?

Reply:
This is to avoid the new repeated syscall type errors to from being also reported as 'Non-explicit control flow...' errors.